artifactory/commands/npm/native.go

```diff
@@ -0,0 +1,115 @@
+package npm
+
+import (
+	"fmt"
+	gofrogcmd "github.com/jfrog/gofrog/io"
+	commandUtils "github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	npmutils "github.com/jfrog/jfrog-cli-core/v2/utils/npm"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+type NativeCommandArgs struct {
+	CommonArgs
+}
+
+type NativeCommand struct {
```

Comment:
Let's add the following description for this struct - 

NativeCommand represents any npm command which is not "install", "ci" or "publish".